stubs/redis/redis/commands/core.pyi

```diff
@@ -137,8 +137,8 @@ class CoreCommands(Generic[_StrType]):
     def zincrby(self, name: _Key, amount: float, value: _Value) -> float: ...
     def zinterstore(self, dest: _Key, keys: Iterable[_Key], aggregate: Literal["SUM", "MIN", "MAX"] | None = ...) -> int: ...
     def zlexcount(self, name: _Key, min: _Value, max: _Value) -> int: ...
-    def zpopmax(self, name: _Key, count: int | None = ...) -> list[_StrType]: ...
-    def zpopmin(self, name: _Key, count: int | None = ...) -> list[_StrType]: ...
+    def zpopmax(self, name: _Key, count: int | None = ...) -> list[tuple[bytes, int]]: ...
+    def zpopmin(self, name: _Key, count: int | None = ...) -> list[tuple[bytes, int]]: ...
```

Comment:
A quick local test shows that the scores are returned as `float` (even if added as `int`), and the key depends on `decode_response`:

```python
In [2]: import redis
In [3]: r = redis.Redis(decode_responses=True)
In [4]: r.zadd("foo", {"ab":1, "de": 1, "xx": 0})
Out[4]: 2
In [5]: r.zpopmax("foo")
Out[5]: [('de', 1.0)]
In [6]: r = redis.Redis()
In [7]: r.zpopmax("foo")
Out[7]: [(b'ab', 1.0)]
```

```suggestion
    def zpopmax(self, name: _Key, count: float | None = ...) -> list[tuple[_StrType, int]]: ...
    def zpopmin(self, name: _Key, count: float | None = ...) -> list[tuple[_StrType, int]]: ...
```

Reply:
Thanks for checking! I wasn't able to get it to work.